webpack.config.js

```diff
@@ -129,6 +129,12 @@ var options = {
         test: /\.(woff|woff2|eot|ttf|otf)$/i,
         type: "asset/resource",
       },
+      {
+        test: /\.m?js/,
+        resolve: {
+          fullySpecified: false
+        }
```

Comment:
@escapedcat what's the status here? 
this still can not be removed?

Reply:
Unfortunately not: https://github.com/aaronbarnardsound/lnmessage/issues/3#issuecomment-1305425021